search-app/src/cmr/search/services/parameters/converters/shapefile.clj

```diff
@@ -74,6 +83,15 @@
           :let [sub-geometry (.getGeometryN geometry index)]]
       (geo/geometry->condition sub-geometry options))))
 
+(defn geometry-point-count
+  "Get the number of points in the given Geometry"
+  [geometry]
```

Comment:
nit: Type hint geometry for readability. Unrelated but i really like this library and wish we would have used it for spatial-lib.

Reply:
Done.